lib/appraisal/bundler_dsl.rb

```diff
@@ -0,0 +1,121 @@
+require 'appraisal/dependency_list'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.